pkg/test/step.go

```diff
@@ -555,6 +556,7 @@ func (s *Step) LoadYAML(file string) error {
 				exKubeconfig := env.Expand(s.Step.Kubeconfig)
 				s.Kubeconfig = cleanPath(exKubeconfig, s.Dir)
 			}
+			s.KubeconfigLoading = s.Step.KubeconfigLoading
```

Comment:
I'm not sure if the kubebuilder markers are respected.
Can you please validate somewhere here that the value is either one of the two allowed values or empty?